examples/sprite_batch/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+set(SRCROOT ${PROJECT_SOURCE_DIR}/examples/sprite_batch)
```

Comment:
`SRCROOT` is not necessary. Just use relative file paths and we can avoid the complexity of creating and using this variable. I recently removed all use of `SRCROOT` in the examples.